bot/cogs/antimalware.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from discord import Message, utils
+from discord.ext.commands import Bot, Cog
+
+from bot.constants import AntiMalware as AntiMalwareConfig, Channels
+
+log = logging.getLogger(__name__)
+
+
+class AntiMalware(Cog):
+    """Cog providing anti-malware behavior."""
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.whitelist = tuple(AntiMalwareConfig.whitelist)
+
+    @Cog.listener()
+    async def on_message(self, message: Message) -> None:
+        """Identify messages with prohibited attachments."""
+        log.trace("Entered AntiMalware.on_message()")
+        rejected_attachments = [a for a in message.attachments if
+                                not a.filename.lower().endswith(self.whitelist)]
+        detected_pyfile = len([a for a in message.attachments if a.filename.lower().endswith('.py')]) > 0
+
+        if len(rejected_attachments) > 0:
```

Comment:
You can use 
```
if rejected_attachments:
      ....
```
instead, because an empty list is `False`

Reply:
Good point.